manage.py

```diff
@@ -19,10 +19,22 @@ def main():
             "available on your PYTHONPATH environment variable? Did you "
             "forget to activate a virtual environment?"
         ) from exc
+
+    if needs_db_connection(sys.argv):
+        ensure_db_connection()
+
     execute_from_command_line(sys.argv)
 
 
-def connect_to_db():
+def needs_db_connection(arguments: list) -> bool:
+    commands = {"migrate": True, "makemigrations": True}
+    for arg in arguments:
```

Comment:
DELET

Reply:
I cannot run any `python manage.py` commands when not in the docker container. I can't even do `python manage.py --help`.

Reply:
Yeah, so you run `docker-compose run <my-app-name> <my-command>`